documentation/modules/exploit/windows/local/lexmark_driver_privesc.md

```diff
@@ -0,0 +1,96 @@
+## Vulnerable Application
+
+[Various Lexmark printer drivers](http://support.lexmark.com/index?page=content&id=TE953) allow escalation of privileges on Windows systems.
```

Comment:
Eh technically according to the advisory isn't it various Lexmark Universal Print Drivers? Also tie this into the following paragraph more to build up a more cohesive story. I've given an example below.

```suggestion
Various Lexmark Universal Print drivers, as listed at the [TE953 advisory page](http://support.lexmark.com/index?page=content&id=TE953) allow local authenticated users to elevate privileges to `SYSTEM` by overwriting the XML file XXXXX, which causes XXXX to happen when a printer driver is installed, thereby elevating user's permissions.
```

Reply:
Created a more detailed explanation at https://github.com/rapid7/metasploit-framework/pull/15519/commits/838142362cc6175cb23fd1be4f8e9470e927a960, but feel free to let me know if this doesn't work. Marking resolved.